src/Tickets/Commerce/Order_Modifiers/Admin/Order_Modifier_Fee_Metabox.php

```diff
@@ -222,8 +222,13 @@ protected function save_ticket_fee( Ticket_Object $ticket, array $raw_data ): vo
 		// Merge the automatic fee IDs into the ticket_order_modifier_fees array.
 		$ticket_order_modifier_fees = array_merge( $raw_data['ticket_order_modifier_fees'], $automatic_fee_ids );
 
+		$fee_ids = [];
 		// Ensure IDs are integers.
-		$fee_ids = array_map( 'absint', $ticket_order_modifier_fees );
+		foreach ( $ticket_order_modifier_fees as $item ) {
+			if ( is_object( $item ) && property_exists( $item, 'id' ) ) {
+				$fee_ids[] = absint( $item->id );
+			}
+		}
```

Comment:
I had to refactor this section because `$ticket_order_modifier_fees` wasn't returning just an array.

It was returning 
```Array
  (
      [0] => stdClass Object
          (
              [id] => 25
              [modifier_type] => fee
              [sub_type] => flat
              [raw_amount] => 5.000000
              [slug] => test-fee
              [display_name] => TEC\Tickets\Commerce\Order_Modifiers\Orders_Test::test_this
              [status] => active
              [created_at] => 2024-11-19 15:03:21
              [start_time] => 
              [end_time] => 
              [updated_at] => 
              [meta_value] => all
          )
  
  )```